src/shadowbox/README.md

```diff
@@ -119,6 +119,17 @@ Remove an access key
 curl --insecure -X DELETE $API_URL/access-keys/2
 ```
 
+Set an access key quota
+(e.g. limit outbound data transfer for access key 2 to 1MB over a 24 hour sliding window)
+```
+curl -v --insecure -X PUT -H "Content-Type: application/json" -d '{"quota": {"quotaBytes": 1000000, "windowHours": 24}}' $API_URL/access-keys/2/quota
+```
+
+Remove an access key quota
+```
+curl -v --insecure -X DELETE $API_URL/access-keys/2/quota
+```
+
```

Comment:
@fortuna seems like here we would want a PUT access-keys/default-quota here.  Do we know that admins want per-key quotas or is it more likely that the ease of the same quota for everyone is what is called for? 

What I'm thinking is this: right now I believe that access key ids are randomly generated.  If that's the case then doing something reasonable like "change everyone's quota" becomes a bit of a pain because you will have to enumerate and iterate over your access keys.  It also requires a linear number of requests as opposed to a single request.

Reply:
IMO, the API should be at the server level: /default-quota. See  https://github.com/Jigsaw-Code/outline-server/pull/456#discussion_r303624942

Reply:
There's plan to have a default quota, but I agree with Alberto that it belongs on the server level.